docs/source/references/config.md

```diff
@@ -1,858 +0,0 @@
-# Iroha Configuration reference
```

Comment:
OK, this file should definitely not be deleted!

Reply:
Oops, I don't know how I did it 😅